website/content/partials/commands/config-source-get_more.mdx

```diff
@@ -0,0 +1,3 @@
+#### Additional Information
```

Comment:
What do you think about instead adding this comment to the end of the help string in the command? 

- https://github.com/hashicorp/waypoint/blob/e73cf791e4d197735fd3136ae338b98c33adb61f/internal/cli/config_source_get.go#L106

That way it appears in both the CLI and at the top of this page, instead of at the bottom where it may be overlooked.

Reply:
I like it, updated!